clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -4291,6 +4291,22 @@ static bool ParseTargetArgs(TargetOptions &Opts, ArgList &Args,
   return Success && Diags.getNumErrors() == NumErrorsBefore;
 }
 
+static void CreateEmptyFile(StringRef HeaderName) {
+  if (HeaderName.empty())
+    return;
+
+  Expected<llvm::sys::fs::file_t> ft = llvm::sys::fs::openNativeFileForWrite(
```

Comment:
```suggestion
  Expected<llvm::sys::fs::file_t> FT = llvm::sys::fs::openNativeFileForWrite(
```
Because we're touching this code anyway...